.travis.yml

```diff
@@ -9,6 +9,9 @@ env:
     #   via the "travis encrypt" command using the project repo's public key
     - secure: "Aca0+L1CYqdRTi5T9aeDM7Es6E9eIbRyFhUlBjqEpSJVkk5mgXWAANuiiqWSr69k7aI5WDiuAZGyGqg/9nkVOaJfiJ1a+AmtN/UEvT0IXmn4nCZupiR4aGHUGYwwY6UD9nvHL92BFx21oTyEnd+XlcHjP0jcwmma0P7fc4xK5bIXPHF/nLs2Xm3aw9iuD6WY7gZ+NZPr/BAcAAc3RQtBWC8hQaTn3V1NDTx7Vj5SolQCDii1lwNSw7L8ozaaKFrkwaJqtClkVqQwujYnP4gnFIURRPiM8sy2mKWlQMj5aOyZP7oEuSYMV5Qe9QS6MYqNc8jSHyswM58s1dVnHtaeL/k5fwwyXKA8eEpmpsy1iNzyru6A36mH94QzTVxqjJO5BeQCo18CveqsxP2Y/HE1JeO/79IRpKBWzB99PDzu5QfQvxLRYzn67QWnzYDGJFUPAkvAiEDFyUglGVWv0Ww67zk+s+UFV62rp8EX9xZxYHab9bWufrqj3Lv8qWMYzP7Fy8yJ/BhmHxxXqDtJWgsq1O3IIiaBjq1ZHGldZe63J6C+F1H5Xa1WyoLqv3CRbfkDOlQh4XGA9FM9tfLAfuVwyfMTc0qPuetGsZuirDIiiHaEb6KXU8YSHVgnjhdJIXx7jcn1DqZ1gHv0nMGZyqF2yjDJaTHMb0lwUedzwOXCXAw="
 
+arch:
+  - ppc64le
```

Comment:
This actually changes all of the entries that don't have an explicit `arch:` attribute to use `ppc64le` instead of `amd64`. So after this patch, it would not be testing x86_64 at all. I've just pushed a more comprehensive version to the master. Thanks for the contribution!